lieer/gmailieer.py

```diff
@@ -861,8 +862,7 @@ def full_pull(self):
             self.bar.total = total
             self.bar_update(len(gids))
 
-            for m in gids:
-                message_gids.append(m["id"])
+            message_gids = [m["id"] for m in gids]
```

Comment:
This always confuses me in python: doesn't this change the scope of message_gids?

Reply:
Yeah, scoping in Python is weird and confusing. But since there is no block scoping in Python, only function scoping, this change does not change the scope of `message_gids`.

Compare to something like:

```
>>> def foo():
...     if True:
...         x = "wow"
...     print(x)
... 
>>> foo()
wow
```

Reply:
Ok, thanks, looks good.